File: app/src/main/java/com/codenicely/services/feelbazaar/vendorapp/dashboard/provider/CourseListProvider.java
package com.codenicely.services.feelbazaar.vendorapp.dashboard.provider;

import com.codenicely.services.feelbazaar.vendorapp.helper.PresenterCallback;

public interface CourseListProvider {
    void requestCourseList(String cookies, PresenterCallback presenterCallback);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/course_list/provider/CourseListProvider.kt
package org.wikiedufoundation.wikiedudashboard.ui.course_list.provider

import org.wikiedufoundation.wikiedudashboard.util.PresenterCallback

interface CourseListProvider {
    fun requestCourseList(cookies: String, presenterCallback: PresenterCallback<*>)
}
